content/services/support/what_can_i_expect.rst

```diff
@@ -33,6 +33,10 @@ What kind of support is included?
 *Odoo Support does not make changes to your production database without your agreement and gives you
 the material and knowledge to do it yourself!*
 
+.. seealso::
+   - :ref:`upgrade/sla`
+   - :doc:`../../administration/maintain/supported_versions`
```

Comment:
For future reference, we can just write the whole path from the parent directory instead of from wherever the file is locally located.
```suggestion
   - :doc:`/administration/maintain/supported_versions`
```